### DISCUSSION TITLE
Connection String for Azure WebPubSub Push Notifications

### DISCUSSION THREAD

**Discussion by konrad79** [shinyorg/shiny#1490] [Q&A] [answered]
I want to receive notifications from my Azure WebPubSub in my Blazor Hybrid App.

As in the example, I add my PushDelegate in the MauiProgram.cs:
https://shinylib.net/client/push/providers/

```
services.AddPushAzureNotificationHubs<PushDelegate>(
    "Your Connection String",
    "Your Hub Name"
);
```

My connection string from Azure looks like:
`"Endpoint=https://....azure.com;AccessKey=123...;Version=1.0;"`

But during startup I get the error:

`System.ArgumentException: 'Invalid URI: The hostname could not be parsed. (Parameter 'connectionString')'`

But what can be wrong with the connection string? Do I only have to provide the Endpoint-URL?

**Comment by aritchie** [comment, marked as answer]
Azure webpubsub is not Azure Notification Hubs.
